src/platform/site-wide/loading-button/LoadingButton.jsx

```diff
@@ -2,16 +2,30 @@ import React from 'react';
 
 export default function LoadingButton({
   isLoading,
+  loadingText,
   children,
   onClick,
   disabled,
   ...props
 }) {
-  const contents = isLoading ? (
-    <i className="fa fa-spinner fa-spin" />
-  ) : (
-    children
-  );
+  let contents;
+  if (loadingText) {
```

Comment:
This looks good to me as a way of not breaking other components.

Something that came to mind just now was maybe having default text (like "loading" or something) that we can display when someone doesn't pass in custom `loadingText`. @1Copenut, what do you think?

Reply:
@1Copenut - Hi Trevor. Did you get a chance to look at this? Thanks